### PR TITLE
Version 0.2

### PR DESCRIPTION
**Pull request by AHReccese**
#### Reference Issues/PRs
#12 #13 #21 
#### What does this implement/fix? Explain your changes.
Version 0.2 release PR

### CODE REVIEW

File: CHANGELOG.md
Comment (sadrasabouri) [inline CHANGELOG.md:52]: The order should be new to old:
```
[Unreleased]: ...
[0.2]: ...
[0.1]: ...
```

**Review by sadrasabouri — CHANGES_REQUESTED** [review]
minor change request.

**Comment by AHReccese** [comment]
Guys, after reviewing it, please do not merge it.

**Review by sadrasabouri — APPROVED** [review]

**Comment by AHReccese** [comment]
I may have a very minor PR (not sure, reviewing every single detail)

**Comment by sadrasabouri** [comment]
Remember to update release with current `dev`.

**Comment by AHReccese** [comment]
> Remember to update release with current `dev`.
@sadrasabouri 
Is it necessary? I think there is no conflict here.

**Comment by sadrasabouri** [comment]
> > Remember to update release with current `dev`.
> > @sadrasabouri
> Is it necessary? I think there is no conflict here.

You can try. But I always wonder merging such a PRs. The `dev` that `release` originated from is not the same `dev` it's merging to.

**Review by sepandhaghighi — APPROVED** [review]
🚀

**Pull request merged**
